apps/omg_api/test/exit_processor/core_test.exs

```diff
@@ -0,0 +1,238 @@
+# Copyright 2018 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.API.ExitProcessor.CoreTest do
+  @moduledoc """
+  Test of the logic of exit processor - not losing exits from persistence, emitting events, talking to API.State.Core
+  """
+  use ExUnitFixtures
+  use ExUnit.Case, async: true
+  use OMG.API.Fixtures
+
+  alias OMG.API.Crypto
+  alias OMG.API.ExitProcessor.Core
+  alias OMG.API.State
+  alias OMG.API.Utxo
+  alias OMG.Watcher.Eventer.Event
+
+  require Utxo
+
+  @eth Crypto.zero_address()
```

Comment:
In the whole codebase, we always declare `@eth Crypto.zero_address()` maybe it is worth to make chore for cleaning this up?

Reply:
I think this is good. `Crypto.zero_address()` is the common function for those apps which depend on `omg_api`. `@eth` is just syntax sugar to shorten stuff and make readable. We have little option to have a 4-char `@eth` which is defined only once and not make a mess.

Also note that `@eth` isn't used in "productioney" code (only tests, perf-test, devhelpers etc), which is good and should remain like this probably. And only code that would otherwise use `Crypto.zero_address` a lot should make this shortcut.